kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -77,7 +77,7 @@
             v-if="isUserLoggedIn && !isLearner && content.num_coach_contents"
             class="coach-footer-icon"
             :value="content.num_coach_contents"
-            :isTopic="isTopic"
+            :isTopic="content.num_coach_contents >= 1"
```

Comment:
This fix does resolve the error, and I also can understand the thinking behind it. It's a bit confusing when reading the code to have `isTopic` seem to be determined by how many pieces of coach content there are, however.

As an alternative, you could use use the `is_leaf` value on the piece of content, since anything that isn't a leaf (the tip/end of the tree) is a topic.

```suggestion
            :isTopic="!content.is_leaf"
```